liveblocks/liveblocks
# Add more update exports

### Description

- Exports LiveObjectUpdate, LiveMapUpdate, and LiveListUpdate types.
- Fixes a typing issue I saw
- Fixes incorrect documentation

#### How to test it

The attached docs example should show you the new update types inside of the switch statement:

![Screenshot 2023-05-09 at 2 21 43 PM](https://github.com/liveblocks/liveblocks/assets/367710/d436d390-a6a1-4384-81b9-28c8dbcc7fba)


#### Related issue(s)
https://github.com/liveblocks/liveblocks/issues/582
